packages/fluentui/react-builder/src/accessibility/AbilityAttributesValidator.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { ATTRIBUTE_NAME_ERROR_ID, setup } from '../ability-attributes/DevEnv';
+
+export type AbilityAttributesErrors = Record<string, Record<string, string>>;
+
+export type AbilityAttributesValidatorProps = {
+  window: Window;
+  onErrorsChanged: (errors: AbilityAttributesErrors) => void;
+};
+
+const getBuilderId = el => {
+  return el ? el.getAttribute('data-builder-id') ?? getBuilderId(el.parentElement) : undefined;
+};
+
+let _lastId = 0;
+
+export const AbilityAttributesValidator: React.FunctionComponent<AbilityAttributesValidatorProps> = ({
+  window,
+  onErrorsChanged,
+}) => {
+  const [errors, setErrors] = React.useState<AbilityAttributesErrors>({});
+
+  React.useMemo(() => {
```

Comment:
Why is `useMemo` used here? Usually `useMemo` is used to remember the value of a computation across multiple render cycles. It doesn't seem like any output value is captured here. What was your intention here?

Was the idea to run this code only once during the initial mount of the component? In that case you can use `useEffect` with an empty dependency array. e.g. 
```tsx
React.useEffect(()=>{
// code goes here
}, []);
```